### PR TITLE
Add kelownaRGB64

### PR DESCRIPTION
**Pull request by Weirdo-F**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kelowna/rgb64/readme.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+![kelownaRGB64](https://i.imgur.com/DzLy87M.png)
+
+A 60% keyboard based on STM32f303
+
+* Keyboard Maintainer:  [Weirdo](https://github.com/Weirdo-F)(https://github.com/Weirdo-F)
+* Hardware Supported: kelownaRGB64
+* Hardware Availability: Not yet
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/kelowna/rgb64/readme.md:11]: Could you add instructions on how to enter the bootloader, so that new firmware can be flashed.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/kelowna/rgb64/readme.md (hunk 1 above)
Comment (Weirdo-F) [reply keyboards/kelowna/rgb64/readme.md:11]: 好的，已经做出修改。

File: keyboards/kelowna/rgb64/rules.mk [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+LAYOUTS = 64_ansi
```
Comment (drashna) [inline keyboards/kelowna/rgb64/rules.mk:24]:
```suggestion
LAYOUTS = 64_ansi

```

Comment (Weirdo-F) [reply keyboards/kelowna/rgb64/rules.mk]: ok thanks！

**Comment by Weirdo-F** [comment]
@drashna

**Review by drashna — APPROVED** [review]

**Pull request merged**